app/controllers/survey_controller.rb

```diff
@@ -1,4 +1,10 @@
 class SurveyController < ApplicationController
   def index
+    @questionList = Question.order(:order)
+    #@questions = [{:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}, {:title => "How large is your hometown?", :answers => ["big", "small", "medium"]}]
+    @questionList.all.each do |theQuestion|
+      theQuestion.answers = Answer.where(question_id: theQuestion.id)
```

Comment:
If you run 'rails console' on the command line within the project you should be able to try that if you load the question.